test/org/sagebionetworks/bridge/models/surveys/TestSurvey.java

```diff
@@ -162,7 +163,7 @@ public static SurveyQuestion selectBy(Survey survey, DataType type) {
     public TestSurvey(boolean makeNew) {
         setGuid(UUID.randomUUID().toString());
         setName("General Blood Pressure Survey");
-        setIdentifier("bloodpressure");
```

Comment:
This is acceptable because we do nothing to assure identifiers are not duplicated. If a researcher uses the same identifier for two different surveys, this breaks nothing except the researchers understanding of their own data. It's like naming two of your kids Sue.